JS/Prototypes/PrototypeAttribute.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>
+        Every JavaScript object has a prototype. The prototype is also an object.
+        
+        All JavaScript objects inherit their properties and methods from their prototype.
+        Prototype Attribute of Objects Created with new Object () or Object Literal</p>
+
+
+
+<script>
```

Comment:
Always create a new file.
Move the below code to a new file and link it on this page.